### PR TITLE
add automation property for salesforce image on and use the textblock in...

### PR DESCRIPTION
**Pull request by helenren**
...stead of the header for host address and host label to make them work for narrator

### CODE REVIEW

**Comment by wmathurin** [comment]
:+1:

File: SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml [hunk 1]
```diff
@@ -134,7 +126,7 @@
                    Text="{Binding Converter={StaticResource Localize}}"
                    Margin="5"
                    Style="{StaticResource HeaderTextBlockStyle}" />
-        <ItemsControl x:Name="ApplicationLogo">
+        <ItemsControl x:Name="ApplicationLogo" AutomationProperties.Name="Salesforce1">
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml]: The property should be more generic, matching more closely the name itself.  It shouldn't be named as a specific application.

File: SalesforceSDK/Salesforce.SDK.Store/Source/Pages/AccountPage.xaml [hunk 2]
```diff
@@ -137,7 +131,7 @@
                    HorizontalAlignment="Center"
                    Margin="5 0 5 20"
                    Style="{StaticResource HeaderTextBlockStyle}" />
-            <ItemsControl x:Name="ApplicationLogo" HorizontalAlignment="Center" VerticalAlignment="Center" MaxWidth="400">
+            <ItemsControl x:Name="ApplicationLogo" HorizontalAlignment="Center" VerticalAlignment="Center" MaxWidth="400" AutomationProperties.Name="Salesforce1">
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Salesforce.SDK.Store/Source/Pages/AccountPage.xaml]: Same. The property should not be named specifically.

File: SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml (hunk 1 above)
Comment (helenren) [inline SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml]: changed to use the application title.

File: SalesforceSDK/Salesforce.SDK.Store/Source/Pages/AccountPage.xaml (hunk 2 above)
Comment (helenren) [inline SalesforceSDK/Salesforce.SDK.Store/Source/Pages/AccountPage.xaml]: changed to use the application title.

**Comment by smcnulty-sfdc** [comment]
Looks good now!

**Pull request merged**